apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetTestUtilities.java

```diff
@@ -45,10 +45,11 @@ public static String keyPrefixForManifest(String location, DataSetManifest manif
    * @param s3Dao the {@link S3Dao} client to use
    * @param bucket the name of the bucket to place the new object in
    * @param manifest the {@link DataSetManifest} to push as an object
+   * @return the uploaded object's s3 key
    */
-  public static void putObject(S3Dao s3Dao, String bucket, DataSetManifest manifest) {
+  public static String putObject(S3Dao s3Dao, String bucket, DataSetManifest manifest) {
     String keyPrefix = keyPrefixForManifest(CcwRifLoadJob.S3_PREFIX_PENDING_DATA_SETS, manifest);
-    putObject(s3Dao, bucket, keyPrefix, manifest);
+    return putObject(s3Dao, bucket, keyPrefix, manifest);
```

Comment:
The S3 keys were being computed in these methods and I needed the actual key for tests.  Now these methods all return the computed key.